app/static/src/app/store/sensitivity/mutations.ts

```diff
@@ -88,5 +89,15 @@ export const mutations: MutationTree<SensitivityState> = {
     },
     [SensitivityMutation.ParameterSetDeleted](state: SensitivityState, parameterSetName: string) {
         delete state.parameterSetResults[parameterSetName];
+    },
+
+    [SensitivityMutation.ParameterSetSwapped](state: SensitivityState, parameterSetName: string) {
+        const temp = state.result;
```

Comment:
use `result` not `temp` here for the name?

Reply:
done! and also changed names for other run mutation temp variables